aspnetcore/blazor/hybrid/handle-errors.md

```diff
@@ -0,0 +1,82 @@
+
+   ![Microsoft Edge DevTools window for a Blazor Hybrid app running on an emulated Pixel 5](~/blazor/hybrid/handle-errors/_static/edge1.png)
+
+:::zone-end
+
+:::zone pivot="ios"
+
+To use Safari developer tools with an iOS app:
+
+1. Open desktop Safari.
+1. Select the **Preferences** > **Advanced** > **Show Develop** menu item.
+1. Run the .NET MAUI Blazor app in the iOS simulator.
+1. Return to desktop Safari. Select **Develop** > **Simulator** > **0.0.0.0**. If multiple entries for **0.0.0.0** are present, select the entry that highlights the `BlazorWebView`. The `BlazorWebView` is highlighted in blue in the iOS simulator when the correct **0.0.0.0** entry is selected.
+
+   ![Safari Develop Simulator open showing two entries for "0.0.0.0" with the second entry selected because it highlights the BlazorWebView in the Visual Studio emulator UI.](~/blazor/hybrid/handle-errors/_static/ios.png)
+
+1. The **Web Inspector** window appears for the `BlazorWebView`.
+1. Developer tools provide a variety of features for working with apps, including which assets the page requested, how long assets took to load, and the content of loaded assets. The following example shows the **Console** tab, which includes any exception messages generated by the framework or developer code:
+
+   ![Safari Web Inspector and Simulator windows for a Blazor Hybrid app running on an emulated iPad mini](~/blazor/hybrid/handle-errors/_static/safari1.png)
+
+:::zone-end
+
+:::zone pivot="macos"
+
+To use Safari developer tools with a macOS app:
+
+1. Open desktop Safari.
+1. Select the **Preferences** > **Advanced** > **Show Develop** menu item.
+1. Run the .NET MAUI Blazor app in the macOS simulator.
+1. Return to desktop Safari. Select **Develop** > **Simulator** > **0.0.0.0**. If multiple entries for **0.0.0.0** are present, select the entry that highlights the `BlazorWebView`. The `BlazorWebView` is highlighted in blue in the macOS simulator when the correct **0.0.0.0** entry is selected.
+1. The **Web Inspector** window appears for the `BlazorWebView`.
+1. Developer tools provide a variety of features for working with apps, including which assets the page requested, how long assets took to load, and the content of loaded assets. The following example shows the **Console** tab, which includes any exception messages generated by the framework or developer code:
+
+   ![Safari Web Inspector for a Blazor Hybrid app](~/blazor/hybrid/handle-errors/_static/safari2.png)
+
+:::zone-end
+
+:::zone pivot="windows"
+
+To use browser developer tools with a Windows app:
+
+1. Start the app with the **Windows Machine** Start button.
+1. Navigate to a page that uses the `BlazorWebView`.
```

Comment:
```suggestion
1. Run the .NET MAUI Blazor app for Windows and navigate to an app page that uses a `BlazorWebView`.
```

To keep consistent with other platforms, I don't think we need to discuss the means by which to start the application here.